pkgs/applications/networking/instant-messengers/mattermost-desktop/default.nix

```diff
@@ -53,7 +53,9 @@ stdenv.mkDerivation {
       --replace /share/mattermost-desktop/mattermost-desktop /bin/mattermost-desktop
 
     makeWrapper '${lib.getExe electron_28}' $out/bin/${pname} \
-      --add-flags $out/share/${pname}/app.asar
+      --set-default ELECTRON_IS_DEV 0 \
+      --add-flags $out/share/${pname}/app.asar \
+      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
```

Comment:
can we construct the string for `NIXOS_OZONE_WL` in a let binding so it's easier to read, all the escaping is quite a bit of obfuscation.